### ISSUE TITLE
[Request] Add PocketStation proper extraction

### ISSUE THREAD

**Issue by TheRadziu** [mmozeiko/pkg2zip#21]
Right now PocketStation is recognized and extracted as PSX title (which is most likely wrong, since its PS Vita app) and produce no files.
```>pkg2zip -x JA0003-PCSC80018_00-POCKETSTATION001.pkg zRIF string
pkg2zip v1.8
[*] loading...
[*] unpacking PSX
[*] unpacking completed
[*] done!
```

**Comment by tomazzz369** [comment]
pkg link for that PocketStation game please

**Comment by TheRadziu** [comment]
edit: apparently its not wrong that its psx type of package (but gameid is Vita), but still needs to be fixed since it does not produce any files, from others extractors it seems to be files inside:
```
[17:14:55] PCSC80018\PS3LOGO.DAT
[17:14:55] PCSC80018\PARAM.SFO
[17:14:55] PCSC80018\ICON0.PNG
[17:14:55] PCSC80018\CONTENT\texture.enc
[17:14:55] PCSC80018\USRDIR\ISO.BIN.EDAT
```

and from a person who had this app installed:
-he found only `ux0:ps1emu/PCSC80018/texture.enc` on his vita

```
http://zeus.dl.playstation.net/cdn/JA0003/PCSC80018_00/pQPQxDhSsKTYgePChkMtuIpyghZywbSOvfKvPsvdKMfgkUPSKGmOvibEysgRQBOURpMTlweKQnpGtFDaIXhaJPbEPRGdFqYUvZprD.pkg
KO5ifR1dQ+d7gRQbg02xABpvAbI/wN/Z2zUkOMQxxNPfDyiIz3+35Bk/e+Tamm67sfhveYJO42iMD24AACpWFJYA
```

**Comment by tomazzz369** [comment]
hmm... realy interesting, for sure this is psx package (PS1emu), but there is no EBOOT.PBP for psv/psp, ISO.BIN.EDAT is for ps3 and we got only texture.enc? i only noticed BOOTABLE parameter is set to no. i have no idea how this app is launched

**Comment by tomazzz369** [comment]
good thing is it's free game you can install and check by yourself

**Comment by TheRadziu** [comment]
yes, this is not bootable app:
https://www.youtube.com/watch?v=LoH9r6xYb38

**Comment by tomazzz369** [comment]
base on this video this is some kind of dlc, you need Mega Man psx to launch PocketStation

**Comment by TheRadziu** [comment]
Yes, they need to be run via PocketStation enabled game, but this is a problem for another day. 
The issue here is just extraction of that one file.
